src/Elmish.WPF/Binding.fs

```diff
@@ -210,23 +211,41 @@ module Binding =
       >> mapMsg ValueOption.toOption
 
 
+  module Cmd =
+
+    let internal createWithParam exec canExec autoRequery =
+      { Exec = exec
+        CanExec = canExec
+        AutoRequery = autoRequery }
+      |> CmdData
+      |> BaseBindingData
+      |> createBinding
+
+    let internal create exec canExec =
+      createWithParam
+        (fun _ -> exec)
+        (fun _ -> canExec)
+        false
+      >> addLazy (fun m1 m2 -> canExec m1 = canExec m2)
+
+  module OneWaySeq =
+
+    open BindingData.OneWaySeq
```

Comment:
Is this only used for the call to `box` on line 240?  If so. I would prefer to remove this and fully qualify that call.p

Even better though would be to rename this function to `boxMinorTypes`. Then I think I am ok with opening the module.

Reply:
Renamed to `boxMinorTypes`